arch/src/x86_64/mod.rs

```diff
@@ -338,6 +341,149 @@ impl CpuidPatch {
     }
 }
 
+pub struct CpuidFeatureEntry {
+    pub function: u32,
+    pub index: u32,
+    pub feature_reg: CpuidReg,
+}
+
+impl CpuidFeatureEntry {
+    fn predefined_feature_entry_list() -> Vec<CpuidFeatureEntry> {
+        vec![
+            // Leaf 0x1, ECX/EDX, feature bits
+            CpuidFeatureEntry {
+                function: 1,
+                index: 0,
+                feature_reg: CpuidReg::ECX,
+            },
+            CpuidFeatureEntry {
+                function: 1,
+                index: 0,
+                feature_reg: CpuidReg::EDX,
+            },
+            // Leaf 0x7, EAX/EBX/ECX/EDX, extended features
+            CpuidFeatureEntry {
```

Comment:
@rbradford To clarify, I added the EAX to the `sub-leaf 0` of leaf 0x7 here (while our previous discussion was from the `sub-leaf 1` of leaf 0x7).